openshift/secrets-template.yaml

```diff
@@ -124,7 +124,59 @@ objects:
   data:
     token: ${SNYK_TOKEN}
     iss: ${SNYK_ISS}
-
+    encryption_key_for_snyk_token: ${ENCRYPTION_KEY_FOR_SNYK_TOKEN}
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: developer-analytics-audit-report-s3
+  type: Opaque
+  data:
+    aws_secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+    aws_access_key_id: ${AWS_ACCESS_KEY_ID}
+    bucket: 
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: cvae-npm-insights-s3
+  type: Opaque
+  data:
+    aws_secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+    aws_access_key_id: ${AWS_ACCESS_KEY_ID}
+    bucket:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: hpf-maven-insights-s3
+  type: Opaque
+  data:
+    aws_secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+    aws_access_key_id: ${AWS_ACCESS_KEY_ID}
+    bucket:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: hpf-pypi-insights-s3
+  type: Opaque
+  data:
+    aws_secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+    aws_access_key_id: ${AWS_ACCESS_KEY_ID}
+    bucket:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: golang-insights-s3
+  type: Opaque
+  data:
+    aws_secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+    aws_access_key_id: ${AWS_ACCESS_KEY_ID}
+    bucket:
```

Comment:
Setting bucket value as suggested before for all components.

Reply:
done